test/widgets/button.spec.ts

```diff
@@ -1,28 +1,19 @@
-import { Feature, UiElement, Widget } from 'concordialang-ui-core'
-import { minify } from 'html-minifier'
-import { fs, vol } from 'memfs'
-import { promisify } from 'util'
-
+import { UiElement} from 'concordialang-ui-core'
 import { Button } from '../../src/widgets/button'
-import WidgetFactory from '../../src/widgets/widget-factory'
 
 
 
 describe('Button', () => {
 
-    const CURRENT_DIR: string = process.cwd()
-
     let buttonUiElement: UiElement | null
     let buttonWidget: Button | null
 
     beforeEach(() => {
```

Comment:
Event `beforeEach` is not necessary, since every test method will probably verify a different facet of the rendering method, and consider a different button.